pkg/controller/certificates/controller.go

```diff
@@ -77,10 +77,13 @@ func New(ctx *controllerpkg.Context) *Controller {
 	ctrl.issuerLister = issuerInformer.Lister()
 	ctrl.syncedFuncs = append(ctrl.syncedFuncs, issuerInformer.Informer().HasSynced)
 
-	clusterIssuerInformer := ctrl.SharedInformerFactory.Certmanager().V1alpha1().ClusterIssuers()
-	clusterIssuerInformer.Informer().AddEventHandler(&controllerpkg.BlockingEventHandler{WorkFunc: ctrl.handleGenericIssuer})
-	ctrl.clusterIssuerLister = clusterIssuerInformer.Lister()
-	ctrl.syncedFuncs = append(ctrl.syncedFuncs, clusterIssuerInformer.Informer().HasSynced)
+	// if scoped to a single namespace
+	if ctx.Namespace == "" {
+		clusterIssuerInformer := ctrl.SharedInformerFactory.Certmanager().V1alpha1().ClusterIssuers()
+		clusterIssuerInformer.Informer().AddEventHandler(&controllerpkg.BlockingEventHandler{WorkFunc: ctrl.handleGenericIssuer})
+		ctrl.clusterIssuerLister = clusterIssuerInformer.Lister()
```

Comment:
I *think* this is okay, because in both `getGenericIssuer` and https://github.com/jetstack/cert-manager/blob/master/pkg/controller/helper.go#L67, we handle the case where the clusterIssuerLister is nil.

Not 100% sure if there are other places that reference the cluster issuer lister - I've gradually been moving those code areas over to use the Helper struct in `pkg/controller` (linked above)